chapter-1-basics.md

```diff
@@ -44,9 +44,17 @@ only the `Assembling` step should be covered by MORYX. These information result
 in the following command:
 
 ```
-$ moryx new PencilFactory --steps Assembling --products GraphitePencil
+$ moryx new PencilFactory --steps Assembling --products GraphitePencil --branch adp
 ```
 
+> NOTE:
+> For this training, the application developer program (ADP), a custom template exists.
+> It is tailored to the scenario and removes some autogenerated code for simplicity and demonstation purposes.
+> By providing the CLI with the branch name 'adp' as parameter it is selected.
```

Comment:
I would remove everything from here. Maybe mention, that the CLI would by default generate a more advanced initial application But I would keep all that away from the training 'cause it is neither relevant nor do we want to train people on the CLI (for now). Maybe a link to the CLI readme would be ok here.